### PR TITLE
TF-2890 Remove calendar event reply description text

### PR DESCRIPTION
**Pull request by tddang-linagora**
### Issue
- #2890

### Resolved
**Before**
Screenshot 2024-05-23 at 10 18 12

**After**
Screenshot 2024-05-23 at 10 18 27

### CODE REVIEW

File: lib/features/email/presentation/widgets/calendar_event/calendar_event_action_banner_widget.dart [hunk 1]
```diff
@@ -73,15 +72,6 @@ class CalendarEventActionBannerWidget extends StatelessWidget {
                   ]
                 )
               ),
-              if (calendarEvent.getSubTitleEventAction(context).isNotEmpty)
-                Text(
-                  calendarEvent.getSubTitleEventAction(context),
-                  style: const TextStyle(
-                    color: AppColor.colorSubTitleEventActionText,
-                    fontSize: CalendarEventActionBannerStyles.subTileTextSize,
```
Comment (dab246) [inline lib/features/email/presentation/widgets/calendar_event/calendar_event_action_banner_widget.dart:81]: Please also remove `colorSubTitleEventActionText ` & `subTileTextSize` if not used

File: lib/l10n/intl_messages.arb [hunk 2]
```diff
@@ -2930,18 +2930,6 @@
...
-    "type": "text",
-    "placeholders_order": [],
-    "placeholders": {}
-  },
-  "subMessageEventActionBannerCanceled": "\"We are canceling the event due to bad weather.\"",
-  "@subMessageEventActionBannerCanceled": {
-    "type": "text",
-    "placeholders_order": [],
-    "placeholders": {}
-  },
```
Comment (dab246) [inline lib/l10n/intl_messages.arb:2944]: Also remove in `intl_language`

Comment (tddang-linagora) [reply lib/l10n/intl_messages.arb:2944]: Done

File: lib/features/email/presentation/widgets/calendar_event/calendar_event_action_banner_widget.dart (hunk 1 above)
Comment (tddang-linagora) [reply lib/features/email/presentation/widgets/calendar_event/calendar_event_action_banner_widget.dart:81]: `AppColor.colorSubTitleEventActionText` is still in used. `CalendarEventActionBannerStyles.subTileTextSize` removed

**Review by dab246 — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**